app/views/layouts/application.html.erb

```diff
@@ -76,7 +76,11 @@
   </li>
   <% alternative_locales.each do |locale| %>
     <li>
-      <%= link_to(t('language', locale: locale), locale: locale) %>
```

Comment:
This is here since the implicit link breaks from moving the `exception_app` to stop using the Rails routes.

Not sure about the value it has to let the users switch to another language from the error pages, I think we can probably remove the link if it is an error page.